klippy/extras/load_cell_probe.py

```diff
@@ -0,0 +1,611 @@
+# Support for bed level probes featuring a load cell at which the hotend is
+# suspended.
```

Comment:
For the top "copyright block" please use just a single one-line comment at the top of the file.  Feel free to add as many comments as you need in a comment block after the top "copyright block".

Thanks,
-Kevin